the_wizz/kdtree_utils.py

```diff
@@ -204,3 +204,175 @@ def get_mean_array(self):
 
     def get_std_array(self):
         return self._std_array
+
+
+class SphericalKDTree(object):
```

Comment:
This mostly looks okay, however, I would keep all of the input/outputs in radians rather than degrees. This will avoid having to constantly convert between the two and possibly eating up time.